configure.ac

```diff
@@ -599,6 +599,7 @@ case $host in
      dnl make sure android stays above linux for hosts like *linux-android*
      TARGET_OS=android
      LEVELDB_TARGET_FLAGS="-DOS_ANDROID"
+     LDFLAGS="$LDFLAGS -static-libstdc++"
```

Comment:
Better to bundle a shared library than to link statically.

Reply:
Google [recommends static](https://developer.android.com/ndk/guides/cpp-support#important_considerations).

Reply:
"If all of your application's native code is contained in a single shared library, ..."

Which it isn't.

Reply:
I assumed that most people will want `bitcoind` without the utils. I don't have strong opinions about this so if you think bundling more than one bin + shared lib is a more common usecase I'll drop this commit.

Reply:
More likely `bitcoin-qt`!